src/services/contracts/safeContracts.ts

```diff
@@ -80,12 +80,8 @@ export const _getMinimumMultiSendCallOnlyVersion = (safeVersion: SafeInfo['versi
   return semver.gte(safeVersion, INITIAL_CALL_ONLY_VERSION) ? safeVersion : INITIAL_CALL_ONLY_VERSION
 }
 
-export const getMultiSendCallOnlyContract = (
-  chainId: string,
-  safeVersion: SafeInfo['version'],
-  provider: Web3Provider,
-) => {
-  const ethAdapter = createEthersAdapter(provider)
+export const getMultiSendCallOnlyContract = (chainId: string, safeVersion: SafeInfo['version']) => {
+  const ethAdapter = createReadOnlyEthersAdapter()
```

Comment:
We already have a function called `getReadOnlyMultiSendCallOnlyContract` which is identical to this one that we can use instead

Reply:
Reverted everything and changed it to use that function instead